features/stream/src/main/java/com/m3u/features/stream/StreamMask.kt

```diff
@@ -316,62 +346,101 @@ internal fun
+                            }
+                        }
+                        Column(
+                            verticalArrangement = Arrangement.Center,
+                            modifier = Modifier
+                                .weight(3f),
+
+                            ) {
+                                Text(
+                                    text = title.trim(),
+                                    style = MaterialTheme.typography.headlineMedium,
+                                    maxLines = 1,
+                                    color = LocalContentColor.current.copy(0.54f),
+                                    fontFamily = FontFamilies.LexendExa,
+                                    textAlign = TextAlign.Center, // Align text horizontally
+                                    modifier = Modifier
+                                        .height(60.dp)
+                                        .basicMarquee()
+                                        .wrapContentHeight(align = Alignment.CenterVertically) // Align text vertically
+                                        .padding(start = if (isLandscape) 0.dp else 16.dp) // Add left padding only in landscape mode
+                                )
+                            }
+                        }
                     }
-                    AnimatedVisibility(
-                        visible = !isPanelExpanded,
-                        enter = fadeIn(),
-                        exit = fadeOut()
+                    Row(modifier = Modifier
+                        .fillMaxWidth()
                     ) {
-                        Text(
-                            text = title.trim(),
-                            style = MaterialTheme.typography.titleMedium,
-                            fontWeight = FontWeight.ExtraBold,
-                            maxLines = 1,
-                            overflow = TextOverflow.Ellipsis,
+                        Column(
+                            horizontalAlignment = Alignment.CenterHorizontally,
                             modifier = Modifier
-                                .basicMarquee()
-                            //                                .sharedElement(
-                            //                                    state = rememberSharedContentState("stream-title"),
-                            //                                    this
-                            //                                )
-                        )
-                    }
+                                .weight(0.5f)
+                                .padding(4.dp)
+
+                        ) {
+                            if (preferences.fullInfoPlayer && (isNew == true || isNew2 == true)) {
+                                CustomTextIcon(
+                                    text = "NEW",
+                                    textColor = Color.White,
+                                    backgroundColor = Color.Blue
```

Comment:
Don't use the hard colors apart from blank and white. Use materialtheme color

Reply:
using hard colors here for indicators on purpose. red for live, blue for new, for easy visibility

Reply:
red can be replaced with materialtheme.color.error or a specific color but not a (255, 0, 0)